flink-read-write-redis/data/expected_output.csv

```diff
@@ -0,0 +1,4 @@
+,item_id,price
```

Comment:
Can you remove the first "index" column from the output for simplicity and consistency with other expected_output files?